unit_tests/test_config.py

```diff
@@ -551,6 +551,14 @@ def test_18_error_if_no_version_repo_ami_selected(self):
 
             self.clear_sct_env_variables()
 
+    def test_19_aws_region_with_no_region_data(self):
+        os.environ['SCT_CLUSTER_BACKEND'] = 'aws'
+        os.environ['SCT_REGION_NAME'] = 'us-east-2'
```

Comment:
ֳNitpick: This test will hopefully be made to fail soon ;-) Wouldn't it be better to try the region name "non-existent-1" (or something equally silly) and be sure that this test continues to pass forever?

Reply:
you got a point here, 
fixed it (and found out my test isn't testing the exception correctly in the process)